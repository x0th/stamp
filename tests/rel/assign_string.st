Object s = "qwe";
Object.s
