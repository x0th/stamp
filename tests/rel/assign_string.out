STDOUT:
[q, w, e]
STDERR:
